designer/client/src/components/modals/ValidationLabels.tsx

```diff
@@ -54,3 +68,64 @@ export default function ValidationLabels(props: Props) {
         </LabelsContainer>
     );
 }
+
+type TemplateLabelProps = {
+    errorCode: string;
+    templateValues: TemplateValues;
+};
+
+const TemplatedType = styled.span({
+    fontWeight: "bold",
+    textDecoration: "underline",
+    "&:hover": {
+        color: "white",
+    },
+});
+
+function I18nTemplateLabel(props: TemplateLabelProps) {
```

Comment:
Maybe we can hide the implementation details and make the component name more generic?